packages/f-checkout/src/components/Checkout.vue

```diff
@@ -374,6 +389,21 @@ $line-height                              : 16px;
 $checkout-width                           : 596px;
 $checkout-padding                         : spacing(x5) 100px;
 
+.c-checkout-switchUserLink {
+  text-align: center;
+
+  a {
+    // TODO: check default link styles in PIE and update fozzie // (should be able to remove these styles then)
```

Comment:
Should add a ticket number to todo's

Reply:
Probably not for open source repos tho.

Reply:
I didn't actually add that, but I completely agree.

Reply:
Yup, please don't add ticket numbers here.